tests/datasets/test_geo.py

```diff
@@ -76,6 +76,15 @@ class TestGeoDataset:
     def dataset(self) -> GeoDataset:
         return CustomGeoDataset()
 
+    @pytest.fixture(scope="class")
+    def files(self) -> list[str]:
```

Comment:
This definitely feels like overkill for what could just be a class variable.

Reply:
It's currently used in only one test, so maybe just move it there even?